scripts/install_grpcurl.sh

```diff
@@ -7,7 +7,7 @@ export HAB_LICENSE="accept-no-persist"
 
 install_grpcurl() {
   #grpcurl implicity depends on latest glibc
```

Comment:
now the comment lies

Reply:
This is true, but I'll fix that in the followup es/pg pin PR